aten/src/ATen/native/TensorAdvancedIndexing.cpp

```diff
@@ -1168,6 +1178,8 @@ TORCH_IMPL_FUNC(scatter_src_out)
  const Tensor& index,
  const Tensor& src,
  const Tensor& out) {
+  dim = maybe_wrap_dim(dim, self.dim());
```

Comment:
Since you've added this same line in `scatter_impl`, I don't think you need it here (same for the other IMPL functinos).

Reply:
Right, my bad. Should be fixed now. Thanks!